### ISSUE TITLE
Question regarding input of drawCylinder.m

### ISSUE THREAD

**Issue by oqilipo** [mattools/matGeom#84]
Hi David

Why does drawCylinder supports the input of multiple Cylinders as cell and not as matrix since the size of one cylinder is 1x7. Hence, multiple cylinders could be stored in Nx7?
```
if iscell(cyl)
    res = zeros(length(cyl), 1);
    for i = 1:length(cyl)
        res(i) = drawCylinder(hAx, cyl{i}, varargin{:});
    end
    
    if nargout > 0
        varargout{1} = res;
    end    
    return;
end
```

Kind regards
oqilipo

**Comment by dlegland** [comment]
Hi,

well, I should have had a very good reason at the time of writing, but I can not remember it(!), and I agree this is not consistent with the rest of the library... 
I can change to numerical array. Is it fine?

regards,
David

**Comment by oqilipo** [comment]
Yes, of course. Thank you!

**Comment by dlegland** [comment]
fixed in 04d9d17b3abc433c1efda4fe782fa02019434f1d!

**Comment by dlegland** [comment]
(with updated doc as well...)
